resources/views/components/wysiwyg.blade.php

```diff
@@ -0,0 +1,10 @@
+{{-- components.wysiwyg
+
+Render open a modal to edit with CKEditor
+
+--}}
+
+<div onclick="Livewire.emit('openModal', 'laravel-views-ckeditor', {{ json_encode(['model' => ['id' => $model->id, 'class' => get_class($model)], 'field' => $field]) }})"
```

Comment:
I think it would be better if this Livewire component were your project's and that somehow you could render it in the table, check the general comment on the PR

Reply:
Im thinking something else, we could implement an `onClickCell` function so you can whatever you need when a cell is clicked, like open up a modal, that way can avoid having a component just for that

Reply:
I can have it just in my app, because I can have a local UI class.
This is just an html version of the editable feature.

But what do you think about the modals? I think they are great for the actions with modals, they can have child actions (child modals).

to add local UI classes is easy, if you don’t like my way to add modal actions, we should have at least an easy way to add  Action buttons, similar to the UI badges, etc.

Reply:
Hi @elfeffe you can do this as it is now, without a local UI class, you can render your own blade components like `UI:: component('components.my-blade-component', ['prop1' => 'value', 'prop2' => 'value'])`, that is what the rest of the UI methods are doing.

About the modals, I'm playing around with the package (I'm refactoring the confirmation modal for the bulk actions) and I really like how the child modals are working, I'll keep trying it out to see what is better, if just open a modal using the alpine `x-show` or adding a more complete modal system.

I liked your idea of having an `UI::actionButton()` method instead of the `onCellClick`, I can add that component to have something with interaction and not just plain text, either way as I mentioned, when you need something more specific, you can render any blade component using the `UI::component()` method, I 'd like to have the same for livewire components like `UI::livewire()`

Reply:
yeah I remember you're having this issue